container/appmaster/cli/dcos-flink/main.go

```diff
@@ -0,0 +1,128 @@
+// Created by @OhRobin
+
+package main
+
+import (
+	"github.com/mesosphere/dcos-commons/cli"
+	"github.com/mesosphere/dcos-commons/cli/client"
+	"gopkg.in/alecthomas/kingpin.v2"
+	"fmt"
+	"log"
+)
+
+
+func main() {
+	app := cli.New()
+
+	handleListSection(app)
```

Comment:
we could make the handler names more descriptive (list what...), but I will change that.